spec/models/course/group_user_spec.rb

```diff
@@ -24,14 +24,5 @@
 
       it { is_expected.not_to be_valid }
     end
-
-    describe '.managers' do
```

Comment:
Since this will be removed, can we remove the redundant `let!` statements as well? (ie. `let!(:course_group_users) ...`)